docs/README.md

```diff
@@ -2,25 +2,31 @@
 
 ## Interacting with the Substra platform
 
-- [Command line interface](../references/cli.md)
-- [SDK](../references/sdk.md)
+- [Command Line Interface](../references/cli.md)
+- [Software Development Kit](../references/sdk.md)
 
 ## Implementing your assets in python
 
-- [Objective base class](https://github.com/SubstraFoundation/substra-tools/blob/dev/docs/api.md#metrics)
-- [Dataset base class](https://github.com/SubstraFoundation/substra-tools/blob/dev/docs/api.md#opener)
-- [Algo base class](https://github.com/SubstraFoundation/substra-tools/blob/master/docs/api.md#algo)
-- [Composite algo base class](https://github.com/SubstraFoundation/substra-tools/blob/master/docs/api.md#compositealgo)
-- [Aggregate algo base class](https://github.com/SubstraFoundation/substra-tools/blob/master/docs/api.md#aggregatealgo)
+To help you implement your assets in Python, please have a look to the [Substra Tools](https://github.com/SubstraFoundation/substra-tools).
 
 ## Learning about the Substra platform
 
-- [Concepts](./concepts.md)
-- [Machine Learning tasks](./ml_tasks.md)
-- [Adding a full pipeline](./full_pipeline_workflow.md)
-- [Adding data samples](./add_data_samples.md)
+- [Concepts](https://doc.substra.ai/platform_description/concepts.html)
```

Comment:
I would rather let only it like: `[Concepts](...) (Objective, Dataset, Algo, Traintuple, ...)` 
And remove the list below (since it is hard to keep it up to date

Reply:
Ok, sure, makes sense! Thank you for your insight!